# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/1961387-2-ref.html
# Commit: 66e788b91a78
# Full Hash: 66e788b91a785ecda515f8a25dc56acab6fc2d63
# Author: Cristina Horotan <chorotan@mozilla.com>
# Date: 2025-05-08 09:24:40
# Regressor Bug: 1961387
# File Overlap Count: 7
# Description:
#   Revert "Bug 1961387 - Add a reftest with larger glyphs, for CBDT version of Noto Color Emoji. r=gfx-reviewers,lsalzman" for causing crashes (bug 1965123)
#   
#   This reverts commit 05d7f427e840f62d7cd9eaffe83df511aad7aa08.
#   
#   Revert "Bug 1961387 - Account for scaling of color-bitmap fonts when setting up the font-unit scale factor. r=gfx-reviewers,lsalzman"
# ==============================================================================

diff -r 82516541b4b1 -r 66e788b91a78 gfx/tests/reftest/1961387-2-ref.html
--- a/gfx/tests/reftest/1961387-2-ref.html	Thu May 08 09:06:49 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,17 +0,0 @@
-<!DOCTYPE html>
-<meta charset="utf-8" />
-<style>
-    div {
-        position: absolute;
-        writing-mode: vertical-lr;
-        font: 250px serif, Noto Color Emoji;
-    }
-    #mask {
-        color: transparent;
-        background: green;
-    }
-</style>
-
-<p>The emoji glyph should not be visible outside the green box:</p>
-
-<div id="mask">.&#x1f400;.</div>